coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/functions/HiveFunctionResolver.java

```diff
@@ -44,6 +44,7 @@
  * Class to resolve hive function names in SQL to Function.
  */
 public class HiveFunctionResolver {
+  private static final String CORAL_VERSIONED_UDF_PREFIX = "coral_udf_version_(\\d+|x)_(\\d+|x)_(\\d+|x)";
```

Comment:
Remove `CORAL_` since `VERSIONED_UDF` is clear enough that it is related to VersionedUDF class?
Should use `CLASS_NAME_PREFIX`?